tethys_apps/templatetags/site_settings.py

```diff
@@ -14,23 +14,27 @@
 @register.filter
 @stringfilter
 def load_custom_css(var):
-    if var.startswith("/"):
-        var = var.lstrip("/")
+    if var.startswith(os.sep):
+        var = var.lstrip(os.sep)
 
     is_file = os.path.isfile(
         os.path.join(settings.STATIC_ROOT, var)
     ) or static_finder.find(var)
 
     if is_file:
-        return '<link href="' + os.path.join("/static", var) + '" rel="stylesheet" />'
+        return (
+            '<link href="'
+            + os.path.join(os.sep, "static", var)
```

Comment:
In this case these are definitely URL paths so this approach won't work. To ensure we are supporting windows we'll need to ensure these are output as Posix Paths.

Reply:
This is what we should be using: `urllib.parse.ulrjoin` https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin